CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css

```diff
@@ -30,3 +30,9 @@
 .hidden#custom {
     -fx-base:#4f7fba;
 }
+
+#options-menu {
+    -fx-background-color: #121212;
```

Comment:
just noting that the dark scheme here has background color applied whereas the light scheme doesn't and instead has padding and color (which dark scheme does not). Do these need to be aligned?

Reply:
This change was based on the options menu in the Data Access View which only needed changes made on the dark mode version. The background color of light mode matches the view whereas the dark mode background was a gradient

Reply:
Got it. Does that include the padding? I would've assumed it being added in one but not the other would mean the options menu is aligned slightly differently depending on whether you're in light or dark mode

Reply:
I've added the padding to both to keep the button the same size